it/controllers/BillsControllerISpec.scala

```diff
@@ -60,23 +71,40 @@ class BillsControllerISpec extends ComponentSpecBase {
         }
       }
 
-      "isAuthorisedUser with an active enrolment, and multiple valid crystallised estimates" should {
+      "isAuthorisedUser with an active enrolment, and multiple valid crystallised bills" should {
 
-        "return the correct page with tax links when calls successful" in {
+        "return the correct page with bills links when calls successful" in {
 
           And("I wiremock stub a successful Income Source Details response with Multiple Business and Property income")
           IncomeTaxViewChangeStub.stubGetIncomeSourceDetailsResponse(testMtditid)(OK, multipleBusinessesAndPropertyResponse)
 
-          And("I wiremock stub a successful Get Last Estimated Tax Liability response")
-          IncomeTaxViewChangeStub.stubGetLatestCalculation(testNino, testYear, taxCalculationCrystallisedResponse)
-          IncomeTaxViewChangeStub.stubGetLatestCalculation(testNino, testYearPlusOne, taxCalculationCrystallisedResponse)
+          And("I stub a successful calculation response for 2017-18")
```

Comment:
Are dates that far back still relevant?

Reply:
it does not matter which year we use, as long as the service brings back the same tax year as the stub one.